tests/unit/core/dynamicBadge.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { dynamicBadge } from "@/core/dynamicData";
+
+describe("Dynamic data badges", () => {
+  describe("#dynamicBadge", () => {
```

Comment:
I've added 3 sections in here.

@dbelokon can you add test cases for the alt text input too?

Reply:
Yeah sure thing! I will follow up next week😄  thanks for reviewing my PR🙏🏻

Reply:
No rush, but a reminder to look at my comments this week / weekend if you can.